models/dfk/dfk_bank_transactions.sql

```diff
@@ -0,0 +1,51 @@
+{{
+	config(
+		materialized='incremental',
+		unique_key='tx_hash',
+		tags=['dfk'],
+		cluster_by=['block_timestamp']
+	)
+}}
+
+with banking_txs as (
+    select
+        tx_hash,
+        block_timestamp,
+        native_from_address as from_address,
+        from_address as from_hex_address,
+        case
+            when data like '0xa59f3e0c%' then 'Deposit'
+            when data like '0x67dfd4c9%' then 'Withdraw'
+        end as type
+    from {{ ref("txs") }} txs
```

Comment:
please split this table into 2 CTES, one for just the 
- select ... from {{ ref(txs) }} where incremental_load_filter
- logics